_resources/port1.0/group/python-1.0.tcl

```diff
@@ -92,8 +92,11 @@ proc python_set_versions {option action args} {
     global name subport python._addedcode
     if {[string match py-* $name]} {
         foreach v [option $option] {
-
-            subport py${v}[string trimleft $name py] { depends_lib-append port:python${v} }
+            if [string match pypy* ${v}] {
+                subport ${v}[string trimleft $name py] { depends_lib-append port:${v} }
```

Comment:
Using `string trimleft` might be overkill since that will try to remove all leading `p` or `y` characters. It works, but `string range` is probably more appropriate.

And, just to be clear, this makes it so it's not always true that `$subport eq "py${python.version}-${python.rootname}"`. I don't know if there are ports relying on that.

Reply:
Fixed.

Reply:
Eh, well, partially, I should add…

> And, just to be clear, this makes it so it's not always true that $subport eq "py${python.version}-${python.rootname}". I don't know if there are ports relying on that.

We either have to break that invariant, or make `${python.version}` someting other than the Python language version. I elected the former, and we can fix individual ports to use the new variables if/when adding PyPy support.